scrapy/mail.py

```diff
@@ -85,11 +85,13 @@ def send(self, to, subject, body, cc=None, attachs=(), mimetype='text/plain', ch
             for attach_name, mimetype, f in attachs:
                 part = MIMEBase(*mimetype.split('/'))
                 part.set_payload(f.read())
+                part.set_charset(charset)
                 Encoders.encode_base64(part)
                 part.add_header('Content-Disposition', 'attachment', filename=attach_name)
                 msg.attach(part)
         else:
             msg.set_payload(body)
+            msg.set_charset(charset)
```

Comment:
This is a no-op, at least in the bug context, as doing this doesn't recode the payload.